### PR TITLE
Add aws session manager windows

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
 - Add aws cli session manager.
**For new tools, please provide total size and installation time.**
### images:
 - [windows19](https://github.visualstudio.com/virtual-environments/_build/results?buildId=74607&view=results)
 - [windows16](https://github.visualstudio.com/virtual-environments/_build/results?buildId=74608&view=results)
#### Related issue:
https://github.com/actions/virtual-environments/issues/1145
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+################################################################################
+##  File:  Validate-AWS-CLI-SM.ps1
+##  Desc:  Validate aws cli session manager
+################################################################################
+
+$sessionMessage = session-manager-plugin
+
+if ($sessionMessage -Match "*plugin was installed successfully*") {
+    Write-Host "$sessionMessage"
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1:9]: Can we put `Write-Host "$sessionMessage"` before `if`. So we won't need to duplicate it.
Also I believe `exit 0` doesn't make more sense.

File: images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+################################################################################
+##  File:  Validate-AWS-CLI-SM.ps1
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1:2]: Do we really need the separate script for validation if we install in the single script?

File: images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1 (hunk 1 above)
Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1:9]: Yes, sure. Should we totally remove if check because if command not found it will fail.

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1]: Done.

File: images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1 (hunk 2 above)
Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1]: Done.

**Pull request merged**